internal/controller/drplacementcontrol_controller.go

```diff
@@ -2467,3 +2478,8 @@ func (r *DRPlacementControlReconciler) drpcHaveCommonClusters(ctx context.Contex
 
 	return drpolicyClusters.Intersection(otherDrpolicyClusters).Len() > 0, nil
 }
+
+func isDRPCSchedulingDisabled(drpc *rmn.DRPlacementControl, ramenConfig *rmn.RamenConfig) bool {
+	value, ok := drpc.GetAnnotations()[ExprimentalFroceSchedulingAnnotation]
+	return ramenConfig.ExperimentalDRPCSchedulingDisable && (!ok || value == "false")
```

Comment:
If the annotation is missing we value will be nil, so we can simplify the check to:

```go
force := drpc.GetAnnotations()[ExprimentalFroceSchedulingAnnotation]
return ramenConfig.ExperimentalDRPCSchedulingDisable && force != "true"
```

But code reads better when using positive terms, so we can use:

```go
func schedulingEnabled(drpc *rmn.DRPlacementControl, ramenConfig *rmn.RamenConfig) bool {
    force := drpc.GetAnnotations()[ExprimentalFroceSchedulingAnnotation]
    return !ramenConfig.ExperimentalDRPCSchedulingDisable || force == "true"
}
```

Reply:
ack